source/core/grids.cc

```diff
@@ -28,30 +28,27 @@ attach_grid_to_triangulation(
     {
       if (mesh_parameters.simplex)
         {
-          // TODO - Serial triangulation should only be created on one
-          // processor... Get communicator from triangulation auto communicator
-          // = triangulation->get_communicator(); auto this_mpi_process =
-          // Utilities::MPI::this_mpi_process(communicator);
-
-          Triangulation<dim, spacedim> basetria(
-            Triangulation<dim, spacedim>::limit_level_difference_at_vertices);
```

Comment:
Will this flag be needed again when/if simplex refinement is available? If yes, we'll have to keep it in mind

Reply:
I am not sure to be honest. The flag  was already there.... Good question. I suggest we leave it at that and just wait?

Reply:
Yes I agree, and for simplex refinement we'll know where to look in case it is needed

Reply:
If you want you can pass in the type of smoothing via the setting parameter.